anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
I guess this change isn't quite related to the commit message (and should have been another commit), as, if I get you correctly, this is just fixing the fact that a hint that only contains a sound currently displays as `[]` instead of `[...]`.
It's also a change that would be more likely accepted, imho (since Anki is supposedly frozen for now)